src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -186,14 +186,56 @@ class TestRequestRateManager : public TestLoadManagerBase,
   ///
   void TestSequences()
   {
-    double request_rate = 200;
-    auto sleep_time = std::chrono::milliseconds(500);
+    stats_->response_delay = std::chrono::milliseconds(10);
+    double request_rate1 = 100;
```

Comment:
Why not take the request rate as set in the `params_` object like test concurrency manager does?

Reply:
concurrency manager uses max_concurrency, which is different than what the real code uses to determine concurrency rate. There is no matching value on the request rate side.  I could have used something else to standardize it, but the two cases are ultimately different.  I may dig into it some more after I fully understand what determines how many threads and how many sequences are created for each of the managers.